mix.exs

```diff
@@ -44,7 +44,7 @@ defmodule Antikythera.Mixfile do
       homepage_url:    @github_url,
       description:     "An Elixir framework to build your own in-house PaaS (Platform as a Service).",
       package:         package(),
-    ] ++ Keyword.update!(Antikythera.MixCommon.common_project_settings(), :docs, &(&1 ++ docs()))
+    ] ++ Keyword.replace!(Antikythera.MixCommon.common_project_settings(), :docs, docs())
```

Comment:
We should avoid overriding "common" settings here.
Now that a9a and gears don't share any `:docs` setting, let's remove `:docs` key in `Antikythera.MixCommon.common_project_settings()` and move
`docs: [output: "exdoc"]`
into both `Antikythera.GearProject` and our antikythera instance's setting.
After that we can simply write as follows:
```ex
[
  ...
  docs: docs(),
  ...
] ++ Antikythera.MixCommon.common_project_settings()